src/main/java/frc/robot/BotSubsystems.java

```diff
@@ -29,8 +29,8 @@ public static void setDefaultCommands() {
         Logger.setup("Roller DefaultCommand -> RollerStop...");
         roller.setDefaultCommand(BotCommands.rollerStop);
 
-        Logger.setup("RollerArm DefaultCommand -> StopRollerArm...");
-        rollerArm.setDefaultCommand(BotCommands.stopRollerArm);
+        Logger.setup("RollerArm DefaultCommand -> LogRollerArmCurrent...");
+        rollerArm.setDefaultCommand(BotCommands.logRollerArmCurrent);
```

Comment:
By making this the default command, the log is going to be spammed with this info.